README.md

```
@@ -37,16 +37,16 @@ python -m mathsys <target> <filename>.math
 
 > [!NOTE]
-> Compiling via command line will also output a `.ltx` file with the *LaTeX.*
+> Targets available are: `watch`, `validate`, `latex`, `web` and `unix-x86-64`.
 
 ## Project Status
 Mathsys is actively developed with regular releases every 1-3 weeks. This project is still in its early stages, so expect major shifts and changes. Most features aren't close to being developed yet.
 
-I say *we* but I'm indeed a solo developer for now, so if you want to get involved learn
+If you want to get involved in the project, [learn how to contribute](https://docs.abscissa.eu).
 
 ## Technical Background
```

markdownlint-cli2 (0.17.2)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)